### PR TITLE
Fix splitWithTail to insert the tail immediately after the outer loop.

### PR DESCRIPTION
**Pull request by resistor**

### CODE REVIEW

File: test/cpp/tensorexpr/test_loopnest.cpp [hunk 1]
```diff
@@ -122,6 +123,52 @@ void testExprSimple02() {
...
+  For* a;
+  For* b;
+  For* c;
+  l.splitWithTail(x_outer, 7, &a, &b, &c);
+
+  Stmt* stmt = l.root_stmt();
+  Stmt* simplified = IRSimplifier::simplify(stmt);
+  Block* body = dynamic_cast<Block*>(simplified);
+  ASSERT_EQ(body->nstmts(), 3);
+  auto biter = body->begin();
```
Comment (nickgg) [inline test/cpp/tensorexpr/test_loopnest.cpp:148]: I suppose the biter is the beginning of the body, that's deep.

File: test/cpp/tensorexpr/test_loopnest.cpp [hunk 2]
```diff
@@ -14,6 +14,7 @@
 #include <torch/csrc/jit/tensorexpr/loopnest.h>
 #include <torch/csrc/jit/tensorexpr/tensor.h>
 #include <torch/csrc/jit/testing/file_check.h>
+#include "torch/csrc/jit/tensorexpr/ir_simplifier.h"
```
Comment (ZolotukhinM) [inline test/cpp/tensorexpr/test_loopnest.cpp:17]: Nit: please use <> and sort includes.

**Review by ZolotukhinM — APPROVED** [review]
Looks good!

File: test/cpp/tensorexpr/test_loopnest.cpp (hunk 2 above)
Comment (resistor) [reply test/cpp/tensorexpr/test_loopnest.cpp:17]: I thought clang-format was supposed to do that for me :-(

**Review by nickgg — APPROVED** [review]
LGTM

**Pull request closed without merging**